contentcuration/contentcuration/frontend/channelList/views/KCard/index.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :to="to"
+  >
+    <div 
+      v-if="layout === 'horizontal'"
+    >
+      <component
+        :is="'div'"
```

Comment:
I think simply just `is="div"` would eliminate the need to pass a wrapped string